### PR TITLE
Fda(needed for other important project, please do not delete)

### PR DESCRIPTION
**Pull request by smilesun**
Cleaned up curve_data2 branch and now there is no conflict

### CODE REVIEW

File: R/FDAClassifPreprocWrapper.R [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#' @title Functional data preproc wrapper
```
Comment (jakob-r) [inline R/FDAClassifPreprocWrapper.R]: add dot at end of to all titles

**Comment by smilesun** [comment]
for this week I need to work on this branch to do some benchmark so tell me before somone wants to touch anything on this branch

**Comment by smilesun** [comment]
bugs to fix
1. multiresolution only support parameter to all channels or seperate parameter for each channel, no other like par = list(fd1=list(len = c(35,46)))

**Comment by pat-s** [comment]
There are so many changes, so many conflicts and no activity recently -> this can be safely closed.

**Pull request closed without merging**